### PR TITLE
Update teams-protected-apis.md

### PR DESCRIPTION
**Pull request by erichui-ms**

### CODE REVIEW

File: concepts/teams-protected-apis.md [hunk 1]
```diff
@@ -8,33 +8,12 @@ ms.prod: "microsoft-teams"
 
 # Protected APIs in Microsoft Teams
 
-Microsoft Teams APIs in Microsoft Graph that access sensitive data are considered protected APIs. These APIs require that you have additional validation beyond permissions and consent before you can use them.
+In the past, Microsoft Teams APIs in Microsoft Graph that accessed sensitive data were considered as protected APIs. App owners were required to submit an approval request through an online form, so that we could approve and track the apps that use the protected APIs.
```
Comment (Danielabom) [inline concepts/teams-protected-apis.md:11]:
```suggestion
```

Comment (Danielabom) [inline concepts/teams-protected-apis.md:11]:
```suggestion
Microsoft Teams APIs in Microsoft Graph that access sensitive data are considered protected APIs. These APIs require that you have additional validation beyond permissions and consent before you can use them.
```

File: concepts/teams-protected-apis.md [hunk 2]
```diff
@@ -8,33 +8,12 @@ ms.prod: "microsoft-teams"
...
 
-The following APIs are currently protected, and all use [Microsoft Graph application permissions](auth/auth-concepts.md#microsoft-graph-permissions):
+The approval process was manual and was not instant.  You spoke and we listened!  We have been building other features that serve a similar purpose.  With features such as [app registration](https://learn.microsoft.com/azure/active-directory/develop/quickstart-register-app) and [subscription association](/graph/metered-api-setup), we are now able to deprecate the manual approval process.
```
Comment (Danielabom) [inline concepts/teams-protected-apis.md:13]:
```suggestion
The following APIs are currently protected, and all use [Microsoft Graph application permissions](auth/auth-concepts.md#microsoft-graph-permissions):
```

File: concepts/teams-protected-apis.md [hunk 3]
```diff
@@ -8,33 +8,12 @@ ms.prod: "microsoft-teams"
...
-
-> [!NOTE]
-> [Send message](/graph/api/channel-post-messages) is not a protected API.
-
-## Request access to protected APIs
-
-To request access to these protected APIs, complete the following [request form](https://aka.ms/teamsgraph/requestaccess). We usually review access requests every Wednesday and deploy approvals every Friday or Monday, except during major holiday weeks in the U.S. Submissions during those weeks are processed the following non-holiday week.
-
-To verify whether your request has been approved, test your application access on the next applicable Monday. If we have additional questions about the request, we will contact the email specified in the form.
+Starting May 18th, 2023, the online form and the protected API approval process are no longer needed.  You can immediately call the protected APIs as long as other requirements ([app registration](https://learn.microsoft.com/azure/active-directory/develop/quickstart-register-app), [subscription association](/graph/metered-api-setup)) are met.
```
Comment (Danielabom) [inline concepts/teams-protected-apis.md:15]:
```suggestion
* [List channel messages](/graph/api/channel-list-messages)
* [Get chatMessages in a channel delta](/graph/api/chatmessage-delta)
* [Get channel message](/graph/api/chatmessage-get)
* [Create subscription for new channel messages](/graph/api/subscription-post-subscriptions)
* [List replies to a message](/graph/api/chatmessage-list-replies)
* [Get a reply to a message](/graph/api/chatmessage-get)
* (List chats doesn't exist, with or without protected API access)
* [List messages in a chat](/graph/api/chat-list-messages)
* [Get message in chat](/graph/api/chatmessage-get)
* [Create subscription for new chat messages](/graph/api/subscription-post-subscriptions)
* [List all hosted content](/graph/api/chatmessage-list-hostedcontents)
* [Get hosted content](/graph/api/chatmessagehostedcontent-get)
* [Get messages in a deleted team](/graph/api/deletedteam-getallmessages)

> [!NOTE]
> [Send message](/graph/api/channel-post-messages) is not a protected API.
## Request access to protected APIs

To request access to these protected APIs, complete the following [request form](https://aka.ms/teamsgraph/requestaccess). We usually review access requests every Wednesday and deploy approvals every Friday or Monday, except during major holiday weeks in the U.S. Submissions during those weeks are processed the following non-holiday week.

To verify whether your request has been approved, test your application access on the next applicable Monday. If we have additional questions about the request, we will contact the email specified in the form.
```

**Comment by Danielabom** [comment]
@erichui-ms to continue with our chat conversation: Conceptual topics showcase the value of your service or feature.⁠ They refer to a high-level or abstract idea or concept related to a technical subject. It focuses on explaining fundamental principles, theories, or frameworks without going into specific implementation details. Also, conceptual and reference content often employ the present tense to support the sense of immediacy, relevance, and value of the topics exposed.

Some of your text would fit in a blog post since it presents a more focused and detailed discussion about a specific aspect of a technical subject. In a very conversational tone, you recollect what happened, how you approached the situation, and what you are doing and will continue to do in the future. There's value in the content but not for a conceptual topic. 

I am tagging my principal content developer, @angelgolfer-ms, and senior content product manager @Lauragra . In case they have something else to add. 

@angelgolfer-ms , @Lauragra:  I updated the PR as suggested, but Eric had a discussion with his team, and they put the content back because of "feedback from multiple stakeholders that they should not actively publicize this change to everyone, so they decided not to do a blog post." Above, you'll find my input.

**Comment by erichui-ms** [comment]
Hi, @Danielabom , @Lauragra , and @angelgolfer-ms .  We are actually planning to remove this page altogether, but because many customers deep link or bookmark this page, we want to add this "deprecation message" to let them know.  Also, in case some customers might mistakenly think our APIs are less secured without this approval, we want to give them more context in the message.

We don't want to publicize this in a blog post to draw unnecessary attention from people who didn't even bookmark this page.    Also, we don't want people who bookmarked this page to file support tickets thinking the page is broken or missing, and they may not even read blog posts.

If it makes everyone comfortable, how about we promise that we will remove this page by thus autumn?

**Comment by erichui-ms** [comment]
Added "ready for content review" label so @Danielabom , @Lauragra , and @angelgolfer-ms can see my question above.  Please comment.  Thanks in advance!

**Comment by erichui-ms** [comment]
Hi, @Danielabom .  It's been almost a month since the change, so we are comfortable with just removing the concepts/teams-protected-apis.md page from the website and from the table of content (TOC).  Could you please help us update and publish the PR?  Thanks in advance!

**Comment by Danielabom** [comment]
Team, @erichui-ms, this is TOC file api-reference/beta/toc.yml; please update it accordingly. Also, remove the files/ you no longer want to have published.

**Comment by erichui-ms** [comment]
Hi, @Danielabom .  I've removed the obsolete page and updated TOC accordingly.  Please help us publish it.  Thanks.

**Review by Danielabom — APPROVED** [review]
PR approved.

File: concepts/metered-api-list.md [hunk 4]
```diff
@@ -10,8 +10,6 @@ ms.custom: scenarios:getting-started
 
 This article provides a list of metered APIs and services in Microsoft Graph. To call these APIs and services, you must associate an active Azure subscription with the calling application. For details, see [Overview of metered APIs and services in Microsoft Graph](metered-api-overview.md). 
 
-Some metered APIs and services in Microsoft Graph are protected and require additional validation beyond permissions and admin consent. Before you can use these protected APIs, you must submit a request.
```
Comment (Lauragra) [inline concepts/metered-api-list.md:13]: If the protected API request no longer applies for Teams APIs, we should remove the Protected API form column from the table in this topic that starts on line 15.

File: concepts/metered-api-list.md [hunk 5]
```diff
@@ -27,5 +25,4 @@ The following table lists the metered APIs and services.
 
 - [Overview of metered APIs and services in Microsoft Graph](/graph/metered-api-overview)
 - [Enable metered APIs and services in Microsoft Graph](/graph/metered-api-setup)
-- [Protected APIs in Microsoft Teams](/graph/teams-protected-apis)
```
Comment (Lauragra) [inline concepts/metered-api-list.md:30]: Please also remove the Protected APIs note from this topic: https://review.learn.microsoft.com/en-us/graph/teams-licenses?branch=pr-en-us-21253

**Comment by Lauragra** [comment]
@erichui-ms , do we need any type of announcement or blog post about this change, since it's not reflected in the changelog, or is it okay to just close down the form and delete the links/references?

**Review by Lauragra — APPROVED** [review]

**Pull request merged**